buildpacks/_shared/shared/command_runner.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2021-2022 VMware, Inc.
+// SPDX-License-Identifier: BSD-2-Clause
+
+package shared
```

Comment:
Placing this in `shared` from the start, because it will be a useful pattern for future buildpacks